VotingApplication/VotingApplication.Web/Api/Controllers/PollController.cs

```diff
@@ -182,5 +236,35 @@ private Poll Create(PollCreationRequestModel pollCreationRequest)
 
             return newPoll;
         }
+
+        private Poll Create2(PollCreationRequestModel2 pollCreationRequest)
```

Comment:
Why is this called `Create2` and not commented out? Surely naming it `Create` is better, especially if we're just commenting code out temporarily?